python/ray/actor.py

```diff
@@ -499,25 +496,26 @@ def _actor_method_call(self,
         worker.check_connected()
 
         function_signature = self._ray_method_signatures[method_name]
-        if args is None:
-            args = []
-        if kwargs is None:
-            kwargs = {}
-        args = signature.extend_args(function_signature, args, kwargs)
+        args = args or []
```

Comment:
Why isn't this inside of `validate_args`?

Reply:
args and kwargs here are also used at L508